server/chat-commands/info.ts

```diff
@@ -1869,7 +1869,7 @@ export const commands: Chat.ChatCommands = {
 
 		if (!this.runBroadcast(`!formathelp ${format ? format.id : target}`)) return;
 
-		if (format) {
+		if (format && format.exists) {
```

Comment:
```suggestion
		if (format?.exists) {
```
surely this should work